www/css/style.css

```diff
@@ -907,56 +923,96 @@ button.button.back-button.buttons.button-clear.header-item {
   text-align: center; padding: 0.25em 0.8em; border-bottom-color: #ccc; border-bottom-width: 0.5px; border-bottom-style: solid;
 }
 .start-time-tag {
-        box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+        width: 68px;
+        height: 30px;
```

Comment:
I am concerned about changing the relative width (16%) to a hardcoded width (68px).
How many form factors have you tested this on?
Does it look reasonable on both iPhone XS and iPhone 13?
What about on android?

Reply:
It looked good on both Android and iPhone yes. I had ran it on an iPad too, and it looked fine. I would love to share some screenshots, but I am unable to retrieve them from my work laptop. Nor can I run Xcode on my personal window computer.

Reply:
then test on different devices sizes on android?

Reply:
Because the font size is hard-coded, and the text is always going to be about `"00:00 XM"`, I do not believe changing the width of the `start-time-tag` from `%` to `px` will make much of a difference, here is one short-example I just tested:

Using percentage:
<img width="938" alt="Screen Shot 2022-09-23 at 11 36 47 AM" src="https://user-images.githubusercontent.com/61334340/192025677-716c0f97-a6fa-43ab-8971-4cd8c127c088.png">

Using pixels:
<img width="945" alt="Screen Shot 2022-09-23 at 11 37 59 AM" src="https://user-images.githubusercontent.com/61334340/192025688-14f100d8-cff7-487a-ab18-bfedfb6d239d.png">

Reply:
@sebastianbarry in the % case the "Displaying xx trips".... text seems to overflow to the next line and look ugly.
Does that also happen with the pixels?

Sorry, other than that one screenshot, nothing else seems to show that part of the screen.

Reply:
The text spilling over is not related to the px vs. percent

It is related to this `ng-if` (you can see when you inspect the element)

We can resolve this conversation now